services/bookstore/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Build the manager binary
+FROM golang:1.14.1 as builder
+
+ARG workDirName=/github.com/aws/aws-service-operator-k8s
```

Comment:
It is a good practice to use `lower_case` variable names. Please change this var to `work_dir`

Reply:
sure updating it.